### DISCUSSION TITLE
p-tuning v2微调以后生成的模型大小有问题，求教

### DISCUSSION THREAD

**Discussion by Duperr** [zai-org/ChatGLM3#1147] [Q&A] [answered]
p-tuning v2微调以后生成的checkpoints-500大小就有16G，这是为什么啊？我用lora微调整个才141M

**Comment by Duperr** [comment]
我发现微调以后p-tuning v2保存的是合并后的模型，而lora保存的是合并前的参数？这是为什么啊？

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
ptuing是完整的整个模型，用了全参的办法
